### PR TITLE
Lighten doc examples by using bobo rather than ZEO

### PR DESCRIPTION
**Pull request by jimfulton**
Two main reasons:

- People can follow examples without needing a compiler

- Make these tests run less slowly.

I considered using fake examples, but I think it's good that people can try the examples and have them work.

I took a stab toward enabling parallel execution using layers, but gave up, but I plan to circle back, because the buildout tests take too long to run.

### CODE REVIEW

**Comment by tseaver** [comment]
LGTM overall.  I would probably edit out the make-hating path integral via `git rebase -i master` before merging (or merge using the "squash" option).

**Comment by jimfulton** [comment]
But I love to hate make :)  But to make you happy....

Hm, what is this squash option...

**Comment by leorochael** [comment]
image

**Comment by jimfulton** [comment]
Thanks. Missed that the button was also a pulldown (I hate make ;)

File: doc/getting-started.rst [hunk 1]
```diff
@@ -754,10 +740,7 @@ details, as well as let you know about features not touched on here.
    rules.
 
 .. [#requirements-one-per-line] Requirements can have whitespace
-   characters as in ``ZEO <=5``, so they're separated by newlines.
-
-.. [#gcc] Currently, this example requires the ability to build
-   Python extensions and requires access to development tools.
+   characters as in ``bobo <3``, so they're separated by newlines.
```
Comment (mgedmin) [inline doc/getting-started.rst]: Is that an intentional :heart:? ;)

Comment (jimfulton) [inline doc/getting-started.rst]: Not consciously :)  If I'd noticed it, I would have fixed the usage above too. :)

**Pull request merged**